src/pages/test-results-search/test-results-search.html

```diff
@@ -59,4 +59,18 @@ <h3>Advanced search search will show here</h3>
     </tab>
   </tabs>
 
+  <div class='results-error'>
+    <p *ngIf="hasSearched && searchResults.length === 0" class="error-text">
+      There were no results found
+    </p>
+    <p *ngIf="hasSearched && searchResults.length === 0" class="error-text">
```

Comment:
do we need two *ngIf s?

Reply:
nice spot - originally they didn't have a div wrapper will update